material-stepper/src/main/res/layout/ms_step_tab.xml

```diff
@@ -27,6 +27,15 @@
             app:srcCompat="@drawable/ic_check"
             android:visibility="gone" />
 
+        <android.support.v7.widget.AppCompatImageView
```

Comment:
I was thinking about that when developing, but I kinda came to realize that even tough it's an indicator and they will never show at the same time, it just seems better to have two separate views, to apply styling, show and hide. 
But it's easily changeable, if you prefer, I'd can do it :)